tasks/model-comparison/main.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from coretex import currentTaskRun, Model
+
+
+def main() -> None:
+    taskRun = currentTaskRun()
+
+    models: list[Model] = taskRun.parameters["models"]
+    logging.info(">> [Model Comparison] Fetching models and starting comparison.")
+
+    if models is None:
```

Comment:
There is no need for this check, models type is marked as `list[Model]` and the parameter is required to exist in task.yaml